xiblint/rules/unavailable_classes.py

```diff
@@ -0,0 +1,32 @@
+from xiblint.rules import Rule
+
+
+class UnavailableClasses(Rule):
+    """
+    Ensures a given custom class isn't used system types are subclassed by a set of classes.
+
+    You must specify a module as part of the class name.
+
+    Example configuration:
+    {
+      "classes": {
+          "SomeModule.LegacyButton": "SomeModule.NewButton"
+      }
+    }
+    """
+    def check(self, context):  # type: (Rule, xiblint.xibcontext.XibContext) -> None
+        unavailable_classes = self.config.get('classes', {})
```

Comment:
probably want a more specific name for the config, it looks like in the docs you put `custom_classes`